app/forms/hyrax/forms/admin_set_form.rb

```diff
@@ -18,11 +17,8 @@ def [](key)
       end
 
       def permission_template
-        PermissionTemplateForm.new(@permission_template)
-      end
-
-      def workflow_name
-        @permission_template.workflow_name
+        template_model = PermissionTemplate.find_by!(admin_set_id: model.id)
+        PermissionTemplateForm.new(template_model)
```

Comment:
This is dead code. The view is no longer using it (this change happened in a different PR). I'm just cleaning up.